scripts/zkai_attention_kv_d64_two_head_seq32_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,1163 @@
+    if any(not isinstance(byte, int) or isinstance(byte, bool) or byte < 0 or byte > 255 for byte in proof):
+        raise AttentionKvD64TwoHeadSeq32FusedSoftmaxTableGateError("fused proof bytes must be uint8 values")
+    return bytes(proof)
+
+
+def fused_artifact_commitments(envelope: dict[str, Any], *, envelope_bytes: bytes) -> tuple[str, str]:
+    try:
+        parsed_envelope = json.loads(envelope_bytes.decode("utf-8"))
+    except (UnicodeDecodeError, json.JSONDecodeError) as err:
+        raise AttentionKvD64TwoHeadSeq32FusedSoftmaxTableGateError(
+            f"fused envelope commitment bytes must decode as JSON: {err}"
+        ) from err
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)